uint32 	msg_counter
float32 SteeringAngle 
